public/stylesheets/style.css

```diff
@@ -151,3 +151,70 @@ input[type="submit"]:hover {
 .post-body {
   margin: 20px;
 }
+
+/* Profile page CSS */
+/* Header secondary classes (grid) */
+
+.profile-content-container {
+  display: grid;
+  grid-template-columns: 25% 50% 20%;
+  border-style: dotted;
+  grid-column-gap: 10px;
+  grid-gap: 10px;
+}
+
+.friends-container {
+  grid-column: 1, 2;
+}
+
+.posts-container {
+  grid-column: 2, 3;
+}
+
+.about-container {
+  grid-column: 3, 4;
+}
+
+/* Header primary classes (flex) */
+.cover-info-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: flex-end;
+}
+
+#profile-photo {
+  flex: 1;
+}
+
+#name-pronouns {
+  flex: 4;
+}
+
+#edit-add-friend-button {
+  flex: 1;
+}
+
+.test {
+  border-style: solid;
+}
```

Comment:
Still needed?

Reply:
Yes

Reply:
Would've left but haven't finished tweaking the layout yet but I know the group was eager for the code